### ISSUE TITLE
Some questions about the code

### ISSUE THREAD

**Issue by graceyangfan** [limx0/nautilus_talks#1]
@limx0  Hello,limx0,very thanks for you share this code,I have read it,but I still have some questions:
(1)In strategy.py#L107 - L100,`self._current_edge` is compute to be positive, but in L129,it is comparing with a negative value?
(2) In strategy.py#L190 I am confused about how target_position and source_position are distinguished，because `self.cache.position(event.position_id)` does not have information about instruments?
(3)In strategy.py#280, I have few knowledge of evaluation pair trading,would you like to provide some reference about `margin_requirements` and `return_bps`?
(4)some APIs about nautilus_trader:what's the difference between `orders_inflight` and `orders_open`?
I'll be glad if you can provide some information about these questions

**Comment by limx0** [comment]
Hey @graceyangfan! Thanks for the questions - I'm away at the moment bhut I'll try and answer these properly in the next day or two!

**Comment by limx0** [comment]
Okay I'm at my computer now: 

1) This looks like a bug; we should either a) let the edge be negative or b) keep track of wanting to go long/short some other way. 

2) In this instance, only events for the source instrument are let through (on #L170), so we know the position is for the source. That being said, this may be problematic if we opened more positions in the source instrument after the target hedge - it could likely use a refactor.

3) Sure! These are basically just used to calculate a more realistic return % based on the margin of the account. When I plugged these two ETFs into my IB account and tried to buy/sell 10k USD worth of each, IB said I had to put up 2.5k and 1.7k USD margin for the two positions, which is why I put

`ib_long_short_margin_requirement = (0.25 + 0.17) / 2.0` 

in the config. Basically I'm saying for each position I put on (long + short), it locks up around 2k of my capital. 

`return_bps` simply means return % in basis points (percentage * 100).  

4) `orders_inflight` are orders that we have sent to the exchange/broker, but have not yet received an acknowledgement for (ie they are submitted, but not yet accepted/rejected). Typically this could be used if you receive data faster than you get responses from the exchange, and you don't want to send a bunch of orders before you receive a confirmation.
`open_orders` are orders that are active (working) In the market.

Hope that answers your questions - let me know if you have any others!
